uwblueprint/childrens-aid-society
# Implement other permitted individuals screen

## Notion ticket link

[Implement Other Permitted Individuals Screen](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* pretty much same/similar implementation to referral forms
* modified some styling / added new styling for dropdowns



## Steps to test
1. checkout this branch
2. on localhost:3000/intake, click next twice to reach the Other Permitted Individuals Screen
3. double check to make sure that the text inputted/dropdown selected updates the information needed
<img width="1126" alt="Screen Shot 2022-07-19 at 10 23 53 PM" src="https://user-images.githubusercontent.com/77185514/179882460-b02d108d-1d83-4f13-9b5e-21f803436c8d.png">



## What should reviewers focus on?
* same as referral forms


## Checklist
